File: app/src/main/java/com/rockspin/bargainbits/data/models/cheapshark/CheapestPriceEver.java
package com.rockspin.bargainbits.data.models.cheapshark;

/**
 * Cheapest price ever.
 */
public final class CheapestPriceEver {

    private String price;
    private Integer date;

    public String getPrice() {
        return price;
    }

    public void setPrice(final String price) {
        this.price = price;
    }

    public Integer getDate() {
        return date;
    }

    public void setDate(final Integer date) {
        this.date = date;
    }
}

File: app/src/main/kotlin/com/rockspin/bargainbits/data/models/CheapestPriceEver.kt
package com.rockspin.bargainbits.data.models

import com.google.gson.annotations.SerializedName

data class CheapestPriceEver(
    @SerializedName("price")
    val price: Double,
    @SerializedName("date")
    val dateTimestamp: Int
)
